src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,50 @@
+package duke.command;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.exception.InvalidInputException;
+import duke.task.Task;
```

Comment:
Hi Wang Mo, I noticed that some imports are not arranged in lexicographical order. For example, 'duke.exception.InvalidInputException' import should be placed before 'duke.ui.Ui' and 'duke.task.Task' import should be placed before 'duke.ui.Ui'.

Reply:
To add on, I like the clear and concise Javadocs you have written. Your code is enjoyable to read as well 👍 .

Reply:
Yes, I am working on that. Thank you so much!